bpf/lib/ipv6.h

```diff
@@ -116,7 +116,7 @@ static __always_inline int ipv6_addr_in_net(const union v6addr *addr,
 				&& (!mask->p4 || ((addr->p4 & mask->p4) == net->p4))))));
 }
 
-#define GET_PREFIX(PREFIX)						\
+#define GET_PREFIX(prefix)						\
 	bpf_htonl(prefix <= 0 ? 0 : prefix < 32 ? ((1<<prefix) - 1) << (32-prefix)	\
```

Comment:
Nit: I think `prefix` should be capitalized instead, to indicate it's a macro argument. That's what we do elsewhere in the C code.

Reply:
Good point, done.